README.md

```diff
@@ -59,6 +65,16 @@ $ rake lint
 
 Running specific tests is not currently supported.
```

Comment:
Please see my comment on the other PR, but yeah, since this seems to be confusing, i'll just take it out https://github.com/Clever/clever-ruby/pull/63#discussion_r41103001